packages/teams-js/src/public/dialog.ts

```diff
@@ -16,21 +18,45 @@ import { runtime } from './runtime';
  *
  * @beta
  */
+export interface Result {
+  err: string;
+  result: string | object;
+}
+
 export namespace dialog {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  type PostMessageChannel = (message: any) => void;
```

Comment:
Should these types also be exported? And if so, do we normally put them in the capability files, or do we have a place specifically for types? (Same question for Result above)

Reply:
I have seen many exported types and interfaces inside capability files. To name a few:  bot, authentication, mail video. There are three interfaces' files for public, private and internal. Not sure what the rule is. 
IS Erin good person to ask about this?

Reply:
and yes, I think, we might need to use the types outside, and need export. I will look more into this